pycon/settings.py

```diff
@@ -212,7 +212,6 @@ def _(x):
 
 MIDDLEWARE = (
     'django.contrib.sessions.middleware.SessionMiddleware',
-    'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
```

Comment:
why this remove?

Reply:
It is not needed anymore with Django 2.0 and later: https://docs.djangoproject.com/en/3.1/releases/2.0/